setup.py

```diff
@@ -13,6 +13,13 @@
     os.system("sudo +x INSTALL")
     os.system("sudo ./INSTALL")
 
+    # keeping this around for proper packaging later
+    # get wheel name
+    #external_files = os.listdir(os.path.join(os.getcwd(), 'external'))
+    #pyside_wheel = [whl for whl in external_files if whl.endswith('.whl') and whl.startswith("PySide2")][0]
+    #depend_links.append(os.path.join(os.getcwd(), 'external', pyside_wheel))
+
+
```

Comment:
Maybe we should open an issue to keep track of packaging related things until we're ready to actually do it